web/src/app/views/about-view/about-view.component.html

```diff
@@ -52,6 +52,43 @@ <h3 class="card-title">Schůzky každý týden</h3>
   </div>
 </section>
 
+<!-- About us -->
+
+<section class="py-4">
+ <div class="container">
+    
+    <div class="row">
+     <div class="col-12 text-center mb-3">
+       <h2>Kdo jsme?</h2>
+     </div>
+   </div>
+   <div class="row align-items-center">
+      <div class="col-12 col-md-7">
+       <h4>Základní informace</h4>
+       <ul class="list-unstyled">
+         <li>Jsme skupina sedmi oddílů s celoroční činností pro kluky i holky od 7 do 15 let. Pro členy pořádáme každý týden v pondělí či ve středu oddílové schůzky, o prázdninách třítýdenní tábor a v průběhu roku široké spektrum akcí, mezi kterými si jistě každý najde, co ho bude bavit.</li>
+       </ul>
+       <h4>Na jaké akce se s námi můžete těšit?</h4>
+       <ul class="list-unstyled">
+         <li>Zaměřujeme se hlavně na vodu pro malé i velké. Pro menší a pro klidnější povahy jsou víkendovky na klasické vodácké řeky, jako je Vltava, Ploučnice či Ohře. Milovníky adrenalinu zas můžeme naučit jezdit na kajaku a vzít je i na divočejší potoky a řeky u nás i v zahraničí, dle jejich schopností. Případně je můžem vzít i třeba do slalomového kanálu v Tróji. Sportovci jistě ocení závody, jako ČPV a Tři jezy, kterých se každoročně účastníme.</li>
+       </ul>
+       <h4>A co když je zima? Nebo co když mě voda tolik nebaví?</h4>
+       <ul class="list-unstyled">
+         <li>Nevadí! Ne všechny naše akce jsou na vodu. V našem programu se najdou pěší výlety, příběhové akce v naší klubovně, akce na kolo a v zimě i lyže a běžky. A pro ty, co se kamarádí s vodou, ale ne se zimou, jsou tu bazény, kde si mohou zaplavat, zapádlovat, a třeba se i naučit eskymovat, přestože venku mrzne až praští.</li>
+        </ul>
+       <h4>Jak se k nám dostat?</h4>
+       <ul class="list-unstyled">
+         <li>Zaujali jsme vás? Výborně! Domluvte se s někým z oddílových vedoucích, nebo se na nás rovnou přijďte podívat v pondělí či středu v 16:00-18:00 do naší loděnice: V náklích 1a, Braník.</li>

```

Comment:
Faktická: jsou to Hodkovičky, nikoliv Braník.
![image](https://user-images.githubusercontent.com/1541727/215041346-46576a31-cece-48e7-ae98-7e546e87e632.png)

Reply:
Vidíš, to jsem si nebyl jistý (a zkontrolovat to tímhle způsobem mě nenapadlo:-)) Opravím